src/utils.js

```diff
@@ -7,25 +9,61 @@ export function extractLaunchParams() {
 }
 
 export function handleLaunch(params) {
+  console.info('handleLaunch', params);
+
   // We use our custom "target" param, since launches with "contentTarget"
   // parameter do not respect "handlesRelaunch" appinfo option. We still
   // fallback to "contentTarget" if our custom param is not specified.
   //
   let { target, contentTarget = target } = params;
+  let href;
+
+  switch (typeof contentTarget) {
+    case 'string': {
+      if (contentTarget.indexOf(YT_BASE_URL.origin) === 0) {
+        console.info('Launching from direct contentTarget');
+        href = contentTarget;
+      } else {
+        // Out of app dial launch with second screen on home: { contentTarget: 'pairingCode=<UUID>&theme=cl&dialLaunch=watch' }
+        console.info('Launching from partial contentTarget');
+        if (contentTarget.indexOf('v=v=') === 0)
+          contentTarget = contentTarget.substring(2);
+
+        href = YT_BASE_URL.toString() + '?' + contentTarget;
+      }
+      break;
+    }
+    case 'object': {
+      console.info(
+        'Voice Launch',
+        contentTarget.intent,
+        contentTarget.intentParam
+      );
 
+      // Ctrl+F tvhtml5LaunchUrlComponentChanged & REQUEST_ORIGIN_GOOGLE_ASSISTANT in base.js for info
+      // TODO: implement google assistant
+      const search = new URLSearchParams();
+      // possible contentTarget.intentParam: PlayContent, SearchContent
+      const actionIsSearch = (contentTarget.intent = 'SearchContent');
```

Comment:
I am not entirely sure, but you probably want `contentTarget.intent == 'SearchContent'` here? I would honestly move this over below into a proper `if(...)` statement, without an intermediate variable

Reply:
Yeah I did mean that. Whoops. Fixing now.